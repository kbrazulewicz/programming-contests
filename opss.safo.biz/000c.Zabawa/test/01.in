4
121
976071668
5104
187876
