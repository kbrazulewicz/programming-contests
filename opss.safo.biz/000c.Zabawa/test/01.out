7
1
184
-1
